Modules/Relationships/src/Relationships.Application/Relationships/Commands/TerminateRelationship/Handler.cs

```diff
@@ -31,11 +30,9 @@ public async Task<TerminateRelationshipResponse> Handle(TerminateRelationshipCom
         relationship.Terminate(_activeIdentity, _activeDevice);
 
         await _relationshipsRepository.Update(relationship);
-
-        var partnerIdentity = relationship.To == _activeIdentity ? relationship.From : relationship.To;
+        _ = relationship.To == _activeIdentity ? relationship.From : relationship.To;
```

Comment:
Wtf indeed, I had no idea something like this could happen... This is how it was at first:
```csharp
        var peer = relationship.To == _activeIdentity ? relationship.From : relationship.To;

        _eventBus.Publish(new RelationshipStatusChangedIntegrationEvent(relationship));
        _eventBus.Publish(new RelationshipTerminatedIntegrationEvent(relationship, peer));
```

Then I removed the second event:
```csharp
        var peer = relationship.To == _activeIdentity ? relationship.From : relationship.To;

        _eventBus.Publish(new RelationshipStatusChangedIntegrationEvent(relationship));
```

And then I ran `dotnet format` to fix the formatting:
```csharp
        _ = relationship.To == _activeIdentity ? relationship.From : relationship.To;

        _eventBus.Publish(new RelationshipStatusChangedIntegrationEvent(relationship));
```

I had no idea `dotnet format` can make such radical changes in code.

Changes made.